examples/solids/index.rst

```diff
@@ -0,0 +1,348 @@
+.. _example-petsc-elasticity:
+
+Solid mechanics elasticity mini-app
+========================================
+
+This example is located in the subdirectory :file:`examples/solids`. It solves
+the steady-state static balance momentum equations using unstructured high-order
+finite element/spectral element spatial discretizations. Moreover, the solid
+mechanics elasticity example has been developed using PETSc, so that the
+pointwise physics (defined at quadrature points) is separated from the
+parallelization and meshing concerns.
+
+
+.. _problem-linear-elasticity:
+
+----------------------------------------
+
+The strong form of the static balance linear momentum at small strain for the three-dimensional linear elasticity problem is given by :cite:`hughes2012finite`:
+
+.. math::
+   :label: lin-elas
+   
+   \nabla \cdot \boldsymbol{\sigma} + \boldsymbol{g} = \boldsymbol{0} 
+
+
+where :math:`\boldsymbol{\sigma}`, :math:`\boldsymbol{\epsilon}`
+and :math:`\boldsymbol{g}` are stress, strain, and forcing functions,
+respectively. Integrating by parts on the divergence term, we arrive at the weak form the of equation :math:numref:`lin-elas`:
+
+.. math::
+
```

Comment:
I would label this equation and reference it later, together with eq. (24) in the first paragraph of the Hyperelasticity at Small Strain section